packages/analytics/dashboard-renderer/src/components/DashboardRenderer.cy.ts

```diff
@@ -60,6 +64,7 @@ describe('<DashboardRenderer />', () => {
       }
     }
 
+    // @ts-ignore
```

Comment:
You shouldn't need to `ts-ignore` this -- can we talk about the error?

Reply:
```
Type 'Promise<{ analytics: true; percentiles: true; api_requests_retention: "24h"; api_requests_retention_ms: number; api_analytics_retention: "30d"; api_analytics_retention_ms: number; }>' is not assignable to type 'Promise<AnalyticsConfig>'.
  Type '{ analytics: true; percentiles: true; api_requests_retention: "24h"; api_requests_retention_ms: number; api_analytics_retention: "30d"; api_analytics_retention_ms: number; }' is not assignable to type 'AnalyticsConfig'.
    Type '{ analytics: true; percentiles: true; api_requests_retention: "24h"; api_requests_retention_ms: number; api_analytics_retention: "30d"; api_analytics_retention_ms: number; }' is not assignable to type 'AnalyticsConfigEnabled'.
      Types of property 'api_requests_retention' are incompatible.
        Type '"24h"' is not assignable to type
```

Reply:
it looks like 24h is not a valid `api_requests_retention` value

```
export type AnalyticsRetention = '1d' | '30d' | '90d' | '180d' | '365d'
export type RequestsRetention = '1d' | '7d' | '14d' | '30d'
```